entitytypes.md

```diff
@@ -78,7 +78,19 @@ A sample can be found at https://dspace7-entities.atmire.com/rest/#https://dspac
 It contains a HAL link to the Relationship Types for the current Item Type (not embedded)
 
 ## Relationship Types for the current Item Type
-**/api/core/itemtypes/<:id>/relationshiptypes**
+**/api/core/entitytypes/<:id>/relationshiptypes**
 
-A sample can be found at https://dspace7-entities.atmire.com/rest/#https://dspace7-entities.atmire.com/rest/api/core/itemtypes/1/relationshiptypes
+A sample can be found at https://dspace7-entities.atmire.com/rest/#https://dspace7-entities.atmire.com/rest/api/core/entitytypes/1/relationshiptypes
 It embeds the [relationshiptypes](relationshiptypes.md) which are linked to the given item type (either on the left or right side)
+
+## Get Entity type from label
+**/api/core/entitytypes/label/<:entity-type-label>**
```

Comment:
Shouldn't this end-point be more like?
`/search/byLabel?label=<:entity-type-label>` like in: https://github.com/DSpace/Rest7Contract/blob/master/relationships.md#relationships-per-relationship-type

I understand the expected result will be the single Entity type record.

Reply:
I agree with @paulo-graca , this new endpoint looks like a `/search` to me as well.  I like Paulo's suggestion here to treat it similar to `/relationships/search/byLabel`

Reply:
I've not used a search here since a search should support multiple results. This endpoint will rather retrieve the entity type based on a different unique property, and can never return more than one result.

This would be similar to using `/api/core/metadataschemas/prefix/dc` as an alternative to `/api/core/metadataschemas/1`. This is why I verified how to best retrieve a single object based on a unique name, and found https://stackoverflow.com/a/20386425

Reply:
@benbosman : Oh, ok. That's right, I forgot that an EntityType's label *must* be unique (even though at a second glance, I see you noted that in the Contract already)

In that case, I agree with you that we don't need a search here. This is fine as-is.  A search is more useful if the results may be more than one.